Tests/AuthenticationTokenDecoderTests.swift

```diff
@@ -29,6 +29,27 @@ class AuthenticationTokenDecoderTests: XCTestCase {
             XCTAssertFalse(configuration.issuer.isEmpty, "The issuer has not been initialized")
             XCTAssertNotNil(configuration.authorization, "The authorization has not been initialized")
             XCTAssertFalse(configuration.authorization!.isEmpty, "The authorization has not been initialized")
+            XCTAssertFalse(configuration.insightsUrl!.isEmpty, "The insightsUrl has not been initialized")
+            XCTAssertFalse(configuration.environment!.isEmpty, "The environment has not been initialized")
+        } catch {
+            XCTFail("should be unexpected error")
+        }
+    }
+
+    func testDecode_valid_configuration_without_optional_fields() {
```

Comment:
rename to testDecode_validConfiguration_withoutInsightsProperties. Rename other test function as well to testDecode_validConfiguration